test/data/jquery-1.9.1.js

```diff
@@ -7036,7 +7036,7 @@ jQuery.extend( {
 				value += "px";
 			}
 
-			// Fixes #8908, it can be done more correctly by specifing setters in cssHooks,
+			// Fixes #8908, it can be done more correctly by specifying setters in cssHooks,
```

Comment:
This is inside a previous release of jQuery used internally so we wouldn't want to change it.